src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-thu calendar--month-length-29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```
The value of a modifier is separated from its name by a double hyphen